Packs/DeCYFIR/Integrations/decyfir/decyfir.yml

```
@@ -0,0 +1,69 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: decyfir
+  version: -1
+configuration:
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue:
+  display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: DeCYFIR API key
+  hidden: false
+  name: decyfir_api_key
+  required: true
+  type: 0
```

```suggestion
- displaypassword: API Key
  name: api_key
  type: 9
  required: true
  hiddenusername: true
  display: ''
```

If no username is needed you can use this configuration instead which is only api_key.

To get this value in the code you can use:
api_key = params.get('api_key').get('password', ''). This way, the api key is hidden which is more secured.